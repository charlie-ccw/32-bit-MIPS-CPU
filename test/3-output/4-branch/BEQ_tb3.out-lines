register_v0_final 0000000d
