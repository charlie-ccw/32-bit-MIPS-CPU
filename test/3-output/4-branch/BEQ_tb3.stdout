RAM : INIT : Loading RAM contents from test/1-binary/4-branch/beq_tb3.txt
TB : INFO : RAM_ACCESS: Read from 0xbfc00000, data: 0x2442000a
TB : INFO : RAM_ACCESS: Read from 0xbfc00004, data: 0x2443000a
TB : INFO : RAM_ACCESS: Read from 0xbfc00008, data: 0x2480000a
TB : INFO : RAM_ACCESS: Read from 0xbfc0000c, data: 0x1044fffc
TB : INFO : RAM_ACCESS: Read from 0xbfc00010, data: 0x24420003
TB : INFO : RAM_ACCESS: Read from 0xbfc00014, data: 0x00000008
TB : INFO : RAM_ACCESS: Read from 0xbfc00018, data: 0x00000000
register_v0_final 0000000d
test/bus_cpu_tb.v:52: $finish called at 470 (1s)
